src/components/SykmeldingViews/OK/APEN/OkApenSykmelding.tsx

```diff
@@ -70,12 +68,22 @@ function OkApenSykmelding({ sykmelding, olderSykmeldingId, olderSykmeldingCount
                 <Spacing>
                     <SykmeldingSykmeldtContainer sykmelding={sykmelding} />
                 </Spacing>
-                {publicEnv.ENABLE_NEW_FORM === 'true' && <SendSykmeldingForm sykmelding={sykmelding} />}
-                <Form sykmelding={sykmelding} />
+                {useNewForm ? <SendSykmeldingForm sykmelding={sykmelding} /> : <Form sykmelding={sykmelding} />}
```

Comment:
Må også skjule `AvbrytPanel` sidan `SendSykmeldingForm` har dette.

```suggestion
                {useNewForm ? <SendSykmeldingForm sykmelding={sykmelding} /> : <><Form sykmelding={sykmelding} /><AvbrytPanel /></>}
```

Reply:
ah ja, søren :sweat_smile:

Reply:
fix'd